### PR TITLE
feat: toast component

### PR DESCRIPTION
**Pull request by tomaszantas**
## Description

Toast component.

### CODE REVIEW

File: src/styles/toast.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+@keyframes fadeOut {
+  from {
+    opacity: 1;
+    transform: translateY(0);
+  }
+  to {
+    opacity: 0;
+    transform: translateY(20px);
+  }
+}
```
Comment (tomaszantas) [inline src/styles/toast.scss:21]: Could we move this into `utils.scss`? This way we have all `@keyframes` in one file

File: src/components/Toast/Toast.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    addToast(toast: ToastProps) {
+      const id = `${Date.now()}-${Math.random()}`;
+      setToasts((prevToasts) => [...prevToasts, { ...toast, id, isExiting: false }]);
+
+      setTimeout(() => {
+        setToasts((prevToasts) => prevToasts.map(t => t.id === id ? { ...t, isExiting: true } : t));
+        setTimeout(() => {
+          setToasts((prevToasts) => prevToasts.filter((t) => t.id !== id));
+        }, 1000);
+      }, (toast.duration || 3000));
```
Comment (tomaszantas) [inline src/components/Toast/Toast.tsx:29]: NIT:

I think 3s is bit too long

```suggestion
      }, (toast.duration || 2000));
```

File: src/components/Toast/Toast.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div className="Layer__toasts-container">
+      {toasts.map((toast) => (
+        <Toast
+          key={toast.id}
+          id={toast.id}
+          content={toast.content}
+          duration={toast.duration}
+          isExiting={toast.isExiting}
+        />
```
Comment (tomaszantas) [inline src/components/Toast/Toast.tsx:42]: NIT:

```suggestion
        <Toast {...toast} />
```

File: src/components/Toast/Toast.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  duration?: number;
+}
+
+const Toast = ({ id, content, isExiting }: ToastProps & { isExiting: boolean }) => (
+  <div id={id} className={classNames('Layer__toast', { enter: !isExiting, exit: isExiting })}>
+    <p>{content}</p>
+  </div>
+);
+
+export const ToastsContainer = forwardRef((props, ref) => {
```
Comment (tomaszantas) [inline src/components/Toast/Toast.tsx:16]: I think `props` is not used, so you can prefix it with underscore `_`
```suggestion
export const ToastsContainer = forwardRef((_props, ref) => {
```

**Review by tomaszantas — COMMENTED** [review]
Looks good in overall. I left few small suggestions. We will need to move toasts into `Layer` context so it's easier to push new notifications from different components.

Also, it would be good to dismiss toast on click.

**Review by ArkuVonSymfon — APPROVED** [review]

**Review by oliwia1003 — APPROVED** [review]

**Pull request merged**